src/gfx/gfx.cpp

```diff
@@ -533,13 +536,55 @@ namespace ari
 			g_mView = _view;
 			g_mViewProj = g_mProj * g_mView;
 		}
-
 		//------------------------------------------------------------------------------
 		sx_mat4 GetViewMatrix()
 		{
 			return g_mView;
 		}
 
+		//------------------------------------------------------------------------------
+		void SetScissorRect(int x, int y, int width, int height, bool origin_top_left)
+		{
+			sg_apply_scissor_rect(x, y, width, height, origin_top_left);
+
+			g_rectScissor.xmin = x;
+			g_rectScissor.ymin = y;
+			g_rectScissor.xmax = width;
+			g_rectScissor.ymax = height;
+		}
+
+		sx_rect GetScissorRect()
+		{
+			return g_rectScissor;
+		}
+		//------------------------------------------------------------------------------
+		void SetViewport(Viewport* p_viewport, bool origin_top_left)
```

Comment:
Rename to SetViewportRect and GetViewportRect

Reply:
fixed